pandas/tests/indexes/datetimes/test_date_range.py

```diff
@@ -525,20 +524,18 @@ def test_freq_divides_end_in_nanos(self):
 
 
 class TestCustomDateRange(object):
```

Comment:
separately, most of these classes could actually be a hierarchy instead to share some code (or could parametrize things), but separate / later PR.